homework/test/test-list.js

```diff
@@ -4,5 +4,6 @@ describe('Test > ', function() {
 
   it('will test some pure functions', function() {
     // Write any pure function assertion here.
+    assert.equal(1+2, 3);
```

Comment:
No. You need to create a pure method in one manager, and then test to invoke it here.